src/course-unit/messages.js

```diff
@@ -21,6 +21,30 @@ const messages = defineMessages({
     id: 'course-authoring.course-unit.xblocks.button.expand-all',
     defaultMessage: 'Expand all',
   },
+  alertMoveSuccessTitle: {
```

Comment:
[nit] Please add description to every translation because it's required in upstream repos.

Reply:
Added, thank you